internal/backends/selector/Cargo.toml

```diff
@@ -21,7 +21,7 @@ eventloop-winit-wayland = ["i-slint-backend-winit/wayland"]
 
 renderer-femtovg = ["i-slint-backend-winit/renderer-femtovg"]
 renderer-skia = ["i-slint-backend-winit/renderer-skia"]
-renderer-software = ["i-slint-backend-winit/renderer-software"]
+renderer-software = ["i-slint-backend-winit?/renderer-software"]
```

Comment:
if it was me, i'd still do this:

```
renderer-software = ["i-slint-core/swrenderer", "i-slint-backend-winit?/renderer-software"]
```

Reply:
I'm curious, why should the backend selector enable the `i-slint-core/swrenderer` feature when it (a) doesn't use anything of `i_slint_core::platform::swrenderer` for its own build and (b) given that `i-slint-backend-winit` has to enable `i-slint-core/swrenderer` in any case (because it uses the API)?